[2025-01-12T13:14:10.688+0000] {processor.py:186} INFO - Started process (PID=717) to work on /opt/airflow/dags/__pycache__/my_first_dag.py
[2025-01-12T13:14:10.689+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/my_first_dag.py for tasks to queue
[2025-01-12T13:14:10.691+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:14:10.691+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/my_first_dag.py
[2025-01-12T13:14:10.700+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:14:10.700+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/my_first_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
SyntaxError: source code string cannot contain null bytes
[2025-01-12T13:14:10.701+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/my_first_dag.py
[2025-01-12T13:14:10.717+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/my_first_dag.py took 0.033 seconds
